docker/batch-init/README.md

```diff
@@ -0,0 +1,124 @@
+This script contains the `--squash` flag, which is optional but recommended. You must enable Docker experimentals on the server if you would like to use this. See [Setup Docker experimentals](#Setup-Docker-experimentals). If your system is not enabled to use squash or you would prefer not to use the `--squash` flag you can remove it from your `build.sh`.
+
+### Execute the build
+
+Once you have updated all the build arguments with your appropriate values, execute the build script with:
+```bash
+$ chmod +x build-scheduler.sh
+$ ./build-scheduler.sh
+```
+
+### Running the container
+
+To run the Batch Scheduler Docker container, copy the `run-scheduler.sh.example` script to a new file, `run-scheduler.sh`. The run script will start the Docker container. Before executing the script, update the passed in Docker environment variables within the `run-scheduler.sh` script. These environment variables should be configured to meet your needs for your particular Batch Initialization execution. Each variable is described in the table below.
+
+| Env Variable               | Description | 
+| ---------------------------|:--------------| 
+| `S3_SUBMISSIONS_BUCKET_PATH`        | The S3 bucket path of the submission archives |       
+| `S3_SUBMISSIONS_RESULTS_DIRECTORY`  | The S3 directory to stage the validation results in `S3_SUBMISSION_BUCKET_PATH`  |   
+| `BATCH_NUM_NODES`        	 | The number of nodes to deploy the AWS Batch job on. *This must match the number of nodes entered when deploying the AWS Batch infrastructure via CloudFormation |  
+| `BATCH_JOB_DEFINITION`       | The AWS Batch job definition to use when executing the validation job |  
+| `BATCH_JOB_QUEUE`     		 | The AWS Batch job queue to use when executing the validation job |  
+| `AWS_SNS_TOPIC_ARN`     	 | The AWS SNS topic to push notifications to during the AWS Batch validation job |  
+
+Execute the run script with:
+```bash
+$ chmod +x run-scheduler.sh
+$ ./run-scheduler.sh
+```
+
+### Setting up Batch Scheduler Cron Job
+
+The Batch Scheduler Docker container can be set up to run via cron job at a specified time. In order to set up a cron job open cron tab with the following command:
+
+```bash
+crontab -e
+```
+
+Once the crontab editor has opened, add the following line and save. You will need to replace `<your-path>` with the the full path to the `run-scheduler.sh` script on your machine. By default, the log of the cron job will be placed in `/tmp/run-scheduler.log`. You can update this path to write the logs to any valid path on your machine. 
+
+Cron is driven by a time specification denoted by the five `*` at the beginning of the command. To specify a time when you would like your `run-scehduler.sh` script to be executed you will need to modify this command. More information on this specification can be found on the [Cron](https://en.wikipedia.org/wiki/Cron) wiki.
```

Comment:
Typo: run-scehduler.sh should be run-scheduler.sh.

Reply:
fixed